src/renderer/components/GroupBase.js

```diff
@@ -11,12 +12,15 @@ const {
   Button
 } = require('@blueprintjs/core')
 
+const DEFAULT_IMAGE = '../images/group_default.png'
```

Comment:
this will break on windows. instead: `path.join(__dirname, '..', 'images', 'group_default.png)`